core/src/utils.ts

```diff
@@ -63,16 +62,30 @@ export function mkTmpFile({ fileName }: { fileName: string }): string {
   return tmpFilePath
 }
 
+let slackSentTime = new Date().getTime()
+let errMsg = null
+
 async function sendToSlack(error) {
-  const exists = await urlExist(SLACK_WEBHOOK_URL)
-  if (exists) {
+  if (SLACK_WEBHOOK_URL) {
     const webhook = new IncomingWebhook(SLACK_WEBHOOK_URL)
     const text = ` :fire: _An error has occurred at_ \`${os.hostname()}\`\n \`\`\`${JSON.stringify(
-      error
+      error[1]
```

Comment:
@joseph-bisonai What is stored in the first field of `error`?

Reply:
From this PR, you started treating it differently than before so I am curious. It could be a bit better to define the value somewhere in advance because it is reused at many places. E.g.:

```typescript
const e = error[1]
```

Reply:
Since the logger is changed to PINO, an error message differs from the previous error message.  PINO logger has another array at index [0], which is unnecessary.  
So I decided to bring only Index[1] contents.

For reusing the value defining a value is okay. I will change it.~

Reply:
Great! Thank you for the explanation!